app/templates/components/wds-global-navigation.hbs

```diff
@@ -21,19 +16,17 @@
 		onQueryChanged=(action 'onSearchQueryChanged')
 	}}
 	{{#if model.user}}
-		{{wds-global-navigation/links
-			dropdownRightAligned=true
-			model=model.user
-			type='user-menu'
-		}}
+		{{wds-global-navigation/user model=model.user}}
 	{{else if model.anon}}
-		{{wds-global-navigation/account-navigation
-			model=model.anon
-		}}
+		{{wds-global-navigation/anon model=model.anon}}
 	{{/if}}
-	<div class="wds-global-navigation__start-a-wiki{{if model.user ' wds-user-is-logged-in'}}">
-		{{#wds-button secondary=true href=model.create_wiki.header.href class="wds-global-navigation__start-a-wiki-button"}}
-			{{i18n model.create_wiki.header.title.key ns='design-system'}}
+
+	<div class="wds-global-navigation__start-a-wiki">
+		{{#wds-button
```

Comment:
I wonder if we could reuse here `wds-global-navigation/link-button` component

Reply:
We have different i18n path here.